src/lang-promql/complete/hybrid.ts

```diff
@@ -330,12 +336,22 @@ export function analyzeCompletion(state: EditorState, node: SyntaxNode): Context
           { kind: ContextKind.Function },
           { kind: ContextKind.Aggregation }
         );
+        if (parent.type.id !== FunctionCallArgs && parent.type.id !== MatrixSelector) {
```

Comment:
Hmm, when I type `n` in a function call parameter, it now doesn't autocomplete `nan`, but should. Is it because of this?

Reply:
ah yeah that's possible. It was more because we don't want to autocomplete the number in the the `sum` for example.
But yeah I forgot that sometimes it's possible ....

Maybe we should just autocomplete it everytime even if it's sometimes false depending of the aggregation / function

Reply:
even if I don't really like it :/.

But well in the other side, putting NaN / Inf is a bit a corner case in the function call parameters right ?

Reply:
Yeah, it's legal in the type system and will cause a valid query, but I'm not aware of any valid use case with the current set of functions taking scalars :) So probably it's fine to just not support it for now, yeah.